.travis.yml

```diff
@@ -38,13 +36,17 @@ install:
   - nvm use $TRAVIS_NODE_VERSION
 
   # Install requirements
-  - pip install -r requirements_devel.txt
+  - pip install -r requirements.txt
+  - pip install coverage
```

Comment:
Any reason we can't install `coverage, pycodestyle, ...` as part of `requirements.txt` ?

Reply:
@collin5, I have finished installing the above-mentioned libraries as part of requirements.txt